22.13.1/Dockerfile

```
@@ -0,0 +1,9 @@
+ARG NODE_VERSION=22.13.1
```

acho que o patch não seria necessário
```suggestion
ARG NODE_VERSION=22.13
```